src/ucp/rndv/proto_rndv.c

```diff
@@ -104,6 +104,12 @@ ucp_proto_rndv_md_map_to_remote(const ucp_proto_rndv_ctrl_init_params_t *params,
 
     ucs_carray_for_each(lane_cfg, ep_config->key.lanes,
                         ep_config->key.num_lanes) {
+        if (lane_cfg->rsc_index == UCP_NULL_RESOURCE) {
+            ucs_assert((lane_cfg - ep_config->key.lanes) ==
+                       ep_config->key.cm_lane);
```

Comment:
IMO this assert is not needed. can just skip the lane.

Reply:
Ok, will remove after CI passes, I want to check if there are any other cases.